app/models/attribute/GlobalAttributeTable.scala

```diff
@@ -242,6 +247,14 @@ object GlobalAttributeTable {
           |    GROUP BY global_attribute.global_attribute_id, gsv_data.gsv_panorama_id
           |) panorama_dates
           |GROUP BY panorama_dates.global_attribute_id"""
+    // Puts set of user_ids associated with the attribute in a comma-separated list in a string
+    val userIds = """SELECT global_attribute.global_attribute_id, array_to_string(array_agg(DISTINCT audit_task.user_id), ',') AS users_list
```

Comment:
For the sake of query efficiency, I think that you should be able to add this to the `imageDates` subquery by adding a join with the `audit_task` table. But let me know if there's something I'm missing here. I also saw that you left joined with this subquery when I would have thought that you could have done a inner join, so maybe I missed something.